yb-voyager/cmd/common.go

```diff
@@ -218,8 +218,8 @@ func getLeafPartitionsFromRootTable() map[string][]string {
 	}
 	tables := msr.TableListExportedFromSource
 	for leaf, root := range msr.SourceRenameTablesMap {
-		leafTable := sqlname.NewSourceNameFromQualifiedName(leaf)
```

Comment:
I forgot exactly what we discussed. Just to re-confirm, this function is only for display purposes, right?

Reply:
Yes right